Frameworks/QuartzCore/CALayer.mm

```diff
@@ -1652,20 +1468,23 @@ - (void)setAffineTransform:(CGAffineTransform)transform {
     newTransform.m[3][0] = transform.tx;
     newTransform.m[3][1] = transform.ty;
 
-    if (memcmp(priv->transform.m, newTransform.m, sizeof(newTransform.m)) == 0) {
-        return;
-    }
-
-    id<CAAction> action = [self actionForKey:_transformAction];
+    [self setTransform:newTransform];
+}
 
-    memcpy(&priv->transform, &newTransform, sizeof(CATransform3D));
-    priv->_frameIsCached = FALSE;
+/**
+ @Status Interoperable
+*/
+- (CGAffineTransform)affineTransform {
+    CGAffineTransform ret;
```

Comment:
>eTransform re [](start = 11, length = 13)

Instead of poking at these directly, you can use `CGAffineTransformMake(a,b,c,d,tx,ty)`